saskatoon/harvest/signals.py

```diff
@@ -37,6 +38,30 @@ def changed_by(sender, instance, **kwargs):
     else:
         instance.changed_by = None
 
+def notify_unselected_pickers(sender, instance, **kwargs):
+    if instance.id:
+        original_instance = sender.objects.get(id=instance.id)
+        if instance.status == "Ready" and original_instance.status == "Date-scheduled":
+            unselected_pickers = instance.get_unselected_pickers()
+            email_list = list()
+            mail_subject = _("[Saskatoon] Not Selected as a picker for upcoming Harvest")
+            for p in unselected_pickers:
+                picker_name = p.picker.name()
+                picker_email = list()
+                picker_email.append(p.picker.email())
+                harvest_details = instance.__str__()
+                message = (_("Hi") + " " + picker_name + ",\n\n" +
+                    _("You have not been selected by the pick leader " +
+                    "to participate in the ") + harvest_details + ". " +
+                    _("You may still be contacted by the pick-leader " +
+                    "if other pickers cancel.") + "\n\n" +
```

Comment:
I would add a note like the following: Your email address is associated to a volunteer profile, re-using the same email next time you request to participate in a pick will prioritize you over pickers that have been participating more over the season.

But this is a suggestion, and @2ynn might want to add 2 cents to that.

In practice, this is decided by the pick leader, and they might choose anyone, but there is formation about this in the guides.